Core/Core/API/APIPostPolicyRequestable.swift

```diff
@@ -22,98 +22,133 @@ public enum PostGradePolicy: String, CaseIterable {
     case everyone, graded
 }
 
-public struct PostAssignmentGradesPostPolicyRequest: APIGraphQLRequestable {
+public struct PostAssignmentGradesInput: Codable, Equatable {
```

Comment:
What would you think about namespacing these Input types within the Request eg `PostAssignmentGradesPostPolicyRequest.Input`? And move its declaration to inside that class?